packages/core/src/credentials-provider/DefaultCredentialProvider.ts

```diff
@@ -0,0 +1,10 @@
+import { CredentialProvider, Credentials } from './CredentialsProvider';
+import { env } from 'process';
+
+export class EnvCredentialProvider implements CredentialProvider {
+  public get(): Promise<Credentials | undefined> {
+    const token = (env as any).BRIGHT_TOKEN;
+
+    return Promise.resolve({ token } as Credentials);
```

Comment:
Typescript implies Duck-Typing, you don't need to cast this object to `Credentials`

```suggestion
    return Promise.resolve({ token });
```